### PR TITLE
Documentation: Add initial etcd3 with rkt docs

### PR DESCRIPTION
**Pull request by dghubble**
rel: #6177
cc: @gyuho @robszumski

### CODE REVIEW

File: Documentation/v3/rkt_guide.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+
+# Running etcd under rkt
+
+The following guide will show you how to run etcd under rkt using the [static bootstrap process](clustering.md#static).
+
+## Running a standalone node
+
+The following rkt run command will expose the rkt client API on port 2379 and expose the peer API on port 2380.
```
Comment (robszumski) [inline Documentation/v3/rkt_guide.md]: **etcd** client API

File: Documentation/v3/rkt_guide.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+```
+
+Trust the CoreOS [App Signing Key](https://coreos.com/security/app-signing-key/).
+
+```
+sudo rkt trust --prefix coreos.com/etcd
+# gpg key fingerprint is: 18AD 5014 C99E F7E3 BA5F  6CE9 50BD D3E0 FC8A 365E
+```
+
+Run the `v3.0.6` version of etcd of specify another release version.
```
Comment (robszumski) [inline Documentation/v3/rkt_guide.md]: typo: or

File: Documentation/v3/rkt_guide.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+
+# Running etcd under rkt
+
+The following guide will show you how to run etcd under rkt using the [static bootstrap process](clustering.md#static).
+
+## Running a standalone node
```
Comment (robszumski) [inline Documentation/v3/rkt_guide.md]: I was confused about this. can you say single node or something like that? or change the description to say "running a single node, which is useful for X or Y"

**Comment by gyuho** [comment]
LGTM after addressing Rob's feedback.

I tried this locally and confirmed that it works.

Thank you! /cc @xiang90

File: Documentation/v3/rkt_guide.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
+
```
Comment (xiang90) [inline Documentation/v3/rkt_guide.md]: remove the empty line here?

Comment (xiang90) [inline Documentation/v3/rkt_guide.md]: actually can we move this into container_guide.md?

File: Documentation/v3/rkt_guide.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+### node3
+
+```
+sudo rkt run --net=default:IP=${NODE3} coreos.com/etcd:v3.0.6 -- -name=node3 -advertise-client-urls=http://${NODE3}:2379 -initial-advertise-peer-urls=http://${NODE3}:2380 -listen-client-urls=http://0.0.0.0:2379 -listen-peer-urls=http://${NODE3}:2380 -initial-cluster=node1=http://${NODE1}:2380,node2=http://${NODE2}:2380,node3=http://${NODE3}:2380
+```
+
+Verify the cluster is healthy and can be reached.
+
+```
+etcdctl --endpoints=http://172.16.28.21:2379 cluster-health
```
Comment (xiang90) [inline Documentation/v3/rkt_guide.md]: use `ETCDCTL_API=3 etcdctl --endpoints=http://172.16.28.23:2379,http://172.16.28.22:2379,http://172.16.28.21:2379 endpoint-health` instead? this is the preferred way to test out an etcd3 cluster.

File: Documentation/v3/rkt_guide.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+Run the `v3.0.6` version of etcd of specify another release version.
+
+```
+sudo rkt run --net=default:IP=${NODE1} coreos.com/etcd:v3.0.6 -- -name=node1 -advertise-client-urls=http://${NODE1}:2379 -initial-advertise-peer-urls=http://${NODE1}:2380 -listen-client-urls=http://0.0.0.0:2379 -listen-peer-urls=http://${NODE1}:2380 -initial-cluster=node1=http://${NODE1}:2380
+```
+
+List the cluster member.
+
+```
+etcdctl --endpoints=http://192.168.1.21:2379 member list
```
Comment (xiang90) [inline Documentation/v3/rkt_guide.md]: ETCDCTL_API=3 etcdctl ...

**Comment by xiang90** [comment]
LGTM

**Comment by xiang90** [comment]
@dghubble Kindly ping. Can you clean this up a little bit? Then we can get this merged this week. Thank you!

**Comment by dghubble** [comment]
Yes, I'll get back to this soon. Apologies.

**Comment by xiang90** [comment]
LGTM. Thank you @dghubble!

**Pull request merged**